modules/core/src/main/java/org/apache/ignite/internal/IgnitionEx.java

```diff
@@ -341,17 +339,15 @@ public static boolean stop(boolean cancel, @Nullable Boolean waitForBackups) {
      *      execution. If {@code false}, then jobs currently running will not be
      *      canceled. In either case, grid node will wait for completion of all
      *      jobs running on it before stopping.
-     * @param waitForBackups If {@code true} then node will wait until all of its data is backed up before shutting
-     *      down. Please note that it will completely prevent last node in cluster from shutting down if any caches
-     *      exist that have backups configured. If {@code null} then
-     *      {@link IgniteSystemProperties#IGNITE_WAIT_FOR_BACKUPS_ON_SHUTDOWN} value is used instead.
+     * @param shutdown If this parameter sets explicitly this policy will use for stopping node.
+     *      If this parameter is {@code null} common cluster policy will use.
      * @param stopNotStarted If {@code true} and node start did not finish then interrupts starting thread.
      * @return {@code true} if named Ignite instance was indeed found and stopped,
      *      {@code false} otherwise (the instance with given {@code name} was
      *      not found).
      */
     public static boolean stop(@Nullable String name, boolean cancel,
-        @Nullable Boolean waitForBackups, boolean stopNotStarted) {
+        @Nullable ShutdownPolicy shutdown, boolean stopNotStarted) {
```

Comment:
Seems that it's also part of shutdown policy. Why not to put it inside ShutdownPolicy?